app/characterization_services/mediainfo_characterization_service.rb

```diff
@@ -152,7 +152,7 @@ def file_object
     # Retrieves the master binary file in this FileSet
     # @return [FileNode]
     def preservation_file
-      if parent.try(:image_resource?)
+      if parent.try(:image_resource?) || parent.try(:media_resource?)
         @file_set.original_file
       else
         @file_set.preservation_file
```

Comment:
Given that the `MediainfoCharacterizationService` is only configured for the media types `audio/mpeg` and `audio/x-wav`, I'm uncertain as to whether this condition would be reached.

Reply:
No, this is not the case, https://github.com/pulibrary/figgy/blob/master/app/characterization_services/mediainfo_characterization_service.rb#L49 will never return `true` if another resource (e. g. `ScannedResource`) is provided with A/V FileSets.